action.yml

```
@@ -7,6 +7,10 @@ inputs:
   path:
     description: 'The file or folder to be cached'
     required: true
+  copy-strategy:
+    description: ''
+    required: true
```

This would be a breaking change. Make optional.
```suggestion
    required: false
```

---

I set the default value to 'move', so if you don't pass it any value, it should use the default (no need to check it in code). Let me know if I should still go ahead and do this check in the code instead and will change